src/model/fxQuotes.js

```diff
@@ -273,6 +492,80 @@ class FxQuotesModel {
     }
   }
 
+  /**
+   * Deals with resends of quote requests (POST) under the API spec:
+   * See section 3.2.5.1, 9.4 and 9.5 in "API Definition v1.0.docx" API specification document.
+   */
+  async handleFxQuoteRequestResend (headers, fxQuoteRequest, span, additionalHeaders) {
+    try {
+      const fspiopSource = headers[ENUM.Http.Headers.FSPIOP.SOURCE]
+      this.writeLog(`Handling resend of fxQuoteRequest: ${util.inspect(fxQuoteRequest)} from ${fspiopSource} to ${headers[ENUM.Http.Headers.FSPIOP.DESTINATION]}`)
+
+      // we are ok to assume the quoteRequest object passed to us is the same as the original...
+      // as it passed a hash duplicate check...so go ahead and use it to resend rather than
+      // hit the db again
+
+      // if we got here rules passed, so we can forward the quote on to the recipient dfsp
+      const childSpan = span.getChild('qs_quote_forwardQuoteRequestResend')
+      try {
+        await childSpan.audit({ headers, payload: fxQuoteRequest }, EventSdk.AuditEventAction.start)
+        await this.forwardFxQuoteRequest(headers, fxQuoteRequest.conversionRequestId, fxQuoteRequest, childSpan, additionalHeaders)
+      } catch (err) {
+        // any-error
+        // as we are on our own in this context, dont just rethrow the error, instead...
+        // get the model to handle it
+        this.log.error(`Error forwarding quote request: ${getStackOrInspect(err)}. Attempting to send error callback to ${fspiopSource}`)
```

Comment:
we can log just this part here:
  ```
     this.log.error('Error forwarding fxQuote request: ', err)
  ```
  coz `this.handleException()` already logs the next info inside itself:
  ```
  async handleException (...) {
    this.log.info('Attempting to send error callback to fspiopSource:', { conversionRequestId, fspiopSource })
    ...
  ```

Reply:
done